qlib/kernel/SignalDef.rs

```diff
@@ -480,16 +532,17 @@ impl fmt::Debug for SigContext {
     }
 }
 
-//TODO  how to set sigcontext
+//TODO support fpstate
 #[cfg(target_arch = "aarch64")]
 impl SigContext {
-    pub fn New(ptRegs: &PtRegs, oldMask: u64, cr2: u64, fpstate: u64) -> Self {
+    pub fn New(ptRegs: &PtRegs, oldMask: u64, cr2: u64, _fpstate: u64) -> Self {
         return Self {
-            fault_address: 0,
+            fault_address: cr2,
```

Comment:
Could you please adjust the naming - [suggested patch](https://paste.sr.ht/blob/1d58b06e9669ed4877dba13fae8a84786713861a)

Reply:
that was a leftover, I'll fix, thx